extensions/wikia/AdEngine/js/template/modal.js

```diff
@@ -64,6 +78,16 @@ define('ext.wikia.adEngine.template.modal', [
 		});
 	}
 
+	function scaleAdIframe(adIframe, params) {
+		var lightBoxHeaderHeight = 40,
+			ratioWidth = win.innerWidth / params.width,
+			ratioHeight = (win.innerHeight - lightBoxHeaderHeight) / params.height,
+			ratio = Math.min(ratioWidth, ratioHeight);
```

Comment:
Let's have additional maximumRatio (set to 2 or 3) to prevent massive scaling of small images.